core/src/test/resources/org/apache/accumulo/core/conf/cluster/2_1_cluster.yaml

```diff
@@ -0,0 +1,39 @@
+#
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+#
+
+manager:
+  - localhost
+
+monitor:
+  - localhost
+
+gc:
+  - localhost
+
+tserver:
+  default:
+    - localhost
+
```

Comment:
Should this also have a compaction-coordinator section?

Reply:
I don't think so, scan servers and external compactions are optional and experimental in 2.1. This is a minimal config for 2.1